openlabcmd/openlabcmd/cli.py

```diff
@@ -54,6 +58,25 @@ def _add_check_cmd(self, parser):
         cmd_check.add_argument('--recover', action='store_true',
                                help='Enable the auto recover mode.')
 
+    def _add_repo_cmd(self, parser):
```

Comment:
Why we need this functionality? Please link this pr with issue(bug or enhancement?)

Reply:
done, see https://github.com/theopenlab/openlab/issues/272